pkg/storage/redis/redis.go

```diff
@@ -228,7 +228,7 @@ func (db *DB) ClearDeleted(ctx context.Context, cutoff time.Time) {
 				{"ZREM": {db.versionSet, id}},
 				{"SREM": {db.deletedSet, id}},
 			}
-			opErr = db.tx(c, cmds)
+			opErr = db.tx(c, cmds) //nolint captured in defer()
```

Comment:
actually I think this is a legit issue. opErr won't be updated in the defer. It would have to be:

```
startTime := time.Now()
defer func() {
    recordOperation(ctx, startTime, "clear_deleted", opErr)
}()
```

https://play.golang.org/p/mm8kW28L_JA

Reply:
Good call.  Thanks.

I think this style work as desired https://play.golang.org/p/1sO_dC18MXn